### PR TITLE
Add election function

### PR DESCRIPTION
**Pull request by tstodter**
WIP because I'd like to add tests and get some feedback on how it's looking right now.

Will add some basic testing instructions.

@SharpNotions/ten-hour-project @andynuge

### CODE REVIEW

**Comment by payers1** [comment]
Any chance we could get an explanation / overview of this in pseudocode? (I know I sound like someone we know right now with a cat and a yoga swing). It's sorta hard to get a grip on this with all the Ramda stuffs. Otherwise, I would say keep going and we'll be able to get a sense of how it works from the tests.

**Comment by tstodter** [comment]
@payers1 Yep, will do. I can also try to remove a lot of the Ramda stuff, if you think that'd be better.

File: src/app.js [hunk 1]
```diff
@@ -19,16 +23,18 @@ router
   .get('*/ping', (ctx, next) => {
     ctx.body = `pong ${new Date().toString()}`;
   })
-  .use(auth)
+  // .use(auth)
```
Comment (tstodter) [inline src/app.js]: oops

**Comment by payers1** [comment]
This is really cool. Given alice's preferences of Lettuce, Bread, Tomato, and bob's preferences of Lettuce, Cheese, Bread `singleTransVoteRankings` returns Lettuce, Bread, Tomato, Cheese and `fiboRankings` returns Lettuce, Bread, Cheese, Tomato. So Lettuce, Bread are 1 and 2 on both lists which is awesome.

**Pull request merged**